### PR TITLE
Knockback: New ui & upgrade to NPM

### PR DESCRIPTION
**Pull request by kmargaritis**
* #1110

### CODE REVIEW

File: examples/knockback/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+    "Backbone.localStorage": "^1.1.3",
```
Comment (sindresorhus) [inline examples/knockback/package.json]: lowercase `b`.

File: examples/knockback/index.html [hunk 2]
```diff
@@ -51,10 +52,10 @@
 		</footer>
 
 		<!-- App Dependencies -->
-		<script src="bower_components/todomvc-common/base.js"></script>
-		<script src="bower_components/jquery/jquery.js"></script>
+		<script src="node_modules/todomvc-common/base.js"></script>
+		<script src="node_modules/jquery/dist/jquery.js"></script>
 		<script src="js/lib/knockback-full-stack-0.16.9.min.js"></script>
-		<script src="bower_components/Backbone.localStorage/backbone.localStorage.js"></script>
+		<script src="node_modules/Backbone.localStorage/backbone.localStorage.js"></script>
```
Comment (sindresorhus) [inline examples/knockback/index.html]: `Backbone.localStorage` = > `backbone.localstorage`

File: examples/knockback/package.json (hunk 1 above)
Comment (sindresorhus) [inline examples/knockback/package.json]: `Backbone.localStorage` = > `backbone.localstorage`

**Comment by kmargaritis** [comment]
:passport_control:

File: examples/knockback/package.json (hunk 1 above)
Comment (kmargaritis) [inline examples/knockback/package.json]: Oh Sindre. Come on! Beers on me if you find again something! I guess you are printing a picture of me to use for Darts! :stuck_out_tongue:

File: examples/knockback/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+	"backbone.localStorage": "^1.1.3",
+	"jquery": "^2.0.0",
+	"todomvc-app-css": "^1.0.0",
+	"todomvc-common": "^1.0.1"
```
Comment (sindresorhus) [inline examples/knockback/package.json]: indentation here is off. should be 2 spaces

Comment (kmargaritis) [inline examples/knockback/package.json]: Sindre there is an issue with that. The .editorconfig uses tabs. Sublime on save uses the .editorconfig so it keeps happening all the time!

Comment (kmargaritis) [inline examples/knockback/package.json]: BTW beers on me!

Comment (sindresorhus) [inline examples/knockback/package.json]: Oh, sorry about that. Fixed: https://github.com/tastejs/todomvc/commit/64484b9e18eabed01b8314af899f50697d167946

**Comment by kmargaritis** [comment]
:passport_control:

**Comment by sindresorhus** [comment]
Can you rebase again? Seems like some more commits got included.

**Comment by kmargaritis** [comment]
Sure thing!

**Comment by kmargaritis** [comment]
:passport_control:

**Comment by kmargaritis** [comment]
@sindresorhus Why the PRs are not seen as merged when actually all the code has been "merged"?

**Comment by sindresorhus** [comment]
@kmargaritis I squash and manually merge them.

**Comment by kmargaritis** [comment]
Was there something that caused trouble? I think that i squashed them all.  (cant check cause i am on mobile)

**Comment by sindresorhus** [comment]
@kmargaritis Nah, just results in a cleaner commit log as it skips the stupid merge commit introduced by pushing the "Merge" button.

**Comment by kmargaritis** [comment]
:+1:

**Pull request closed without merging**